examples/grib_log.py

```diff
@@ -0,0 +1,60 @@
+import bokeh.io
```

Comment:
This file should not be included, as this functionality exists in bokeh_garden/logging_bg.py above